### ISSUE TITLE
Large amount of text (5MB) in context takes a lot of time for conversion when run_squad.py is executed

### ISSUE THREAD

**Issue by JeevaTM** [google-research/bert#751]
I ran the following command line:
```
python run_squad.py \
  --vocab_file=$BERT_LARGE_DIR/vocab.txt \
  --bert_config_file=$BERT_LARGE_DIR/bert_config.json \
  --init_checkpoint=$BERT_LARGE_DIR/model.ckpt \
  --do_train=False \
  --train_file=$SQUAD_DIR/train-v2.0.json \
  --do_predict=True \
  --predict_file=$SQUAD_DIR/dev-v2.0.json \
  --train_batch_size=24 \
  --learning_rate=3e-5 \
  --num_train_epochs=2.0 \
  --max_seq_length=384 \
  --doc_stride=128 \
  --output_dir=gs://some_bucket/squad_large/ \
  --use_tpu=True \
  --tpu_name=$TPU_NAME \
  --version_2_with_negative=True
```
dev-v2.0.json has json strcuture as follows:
```
{
  "data": [
    {
      "paragraphs": [
        {
          "qas": [
            {
              "question": "question",
              "id": "65432sd54654dadaad"
            }
          ],
          "context": "paragraph"
        }
      ]
    }
  ]
}
```
When I ask the question for the same context, it is converted to 1 0. If context only has a single paragraph then it takes a minute for each question to get the predictions.json in the output directory.
If the context is too large, it takes hours to run for each question. Where is the conversion happening? So I can store the context to converted data and use it for prediction instead of trying convert everytime.

**Comment by JeevaTM** [comment]
I see the convert_example_to_feature function is generating those masks, index, etc. Is it possible to store those and use it for different questions?
